### DISCUSSION TITLE
Measurable Habits?

### DISCUSSION THREAD

**Discussion by nirmaljpatel** [daya0576/beaverhabits#44] [Ideas]
Let me begin with a compliment that this looks like a great beginning and just love the clean UX.

Is there a plan to add Measurable Habits? 
Habit: Pushups with ability to add how many I actually did.
Loop supports these and seems like a must-have for a habit tracker app.

**Comment by daya0576** [comment]
Yes, that's something missing that should definitely be implemented next!
